extensions/wikia/TemplateClassification/TemplateClassification.hooks.php

```diff
@@ -175,8 +175,8 @@ public function onEditPageLayoutExecute( \EditPageLayoutController $editPage ) {
 		$user = $editPage->getContext()->getUser();
 		$title = $editPage->getContext()->getTitle();
 		if ( ( new Permissions() )->shouldDisplayEntryPoint( $user, $title ) ) {
-			$editPage->addExtraPageControlsHtml(
-				( new View )->renderEditPageEntryPoint( $wgCityId, $title, $user )
```

Comment:
I think that was the only use of this method
https://github.com/Wikia/app/blob/CE-2955/extensions/wikia/TemplateClassification/TemplateClassificationView.php#L87-87
so we can remove it.